src/bootstrap/format.rs

```diff
@@ -0,0 +1,30 @@
+//! Runs rustfmt on the repository.
+
+use crate::{util, Build};
+use std::path::PathBuf;
+use std::process::Command;
+
+pub fn format(build: &Build, check: bool) {
+    let target = &build.build;
+    let rustfmt_var = match std::env::var("RUSTFMT") {
+        Ok(rf) if rf.len() > 0 => rf,
```

Comment:
I'd prefer that the environment variable is just not set entirely versus set to an empty string -- i.e., we shouldn't try to detect an empty string here.